package.json

```diff
@@ -33,26 +33,37 @@
   "contributes": {
     "iconThemes": [
       {
-        "id": "glyph.legacy",
+        "id": "glyph.legacy-original",
         "label": "%iconThemeLabelLegacy%",
-        "path": "theme/legacy-icon-theme.json"
+        "path": "./dist/fileicons/legacy-original-icon-theme.json"
       },
       {
-        "id": "glyph.minimal",
+        "id": "glyph.legacy-minimal",
         "label": "%iconThemeLabelMinimal%",
-        "path": "theme/minimal-icon-theme.json"
+        "path": "./dist/fileicons/legacy-minimal-icon-theme.json"
       }
     ]
   },
   "scripts": {
-    "prepare": "husky"
+    "prepare": "husky",
+    "build:fonts": "fantasticon",
+    "build:dist": "tsc",
+    "build:iconsets": "node ./dist/scripts/build-iconsets.js",
+    "build": "npm-run-all -s build:*"
+  },
+  "dependencies": {
+    "lodash": "^4"
```

Comment:
maybe this shouldn't be a devDep instead of a dep (for now). The implementation for commands/settings isn't there yet. No need to bloat it